plonk-wasm/src/srs.rs

```diff
@@ -320,6 +320,33 @@ pub mod fp {
         });
         basis.iter().map(Into::into).collect()
     }
+
+    /// Web-compatible version that returns pointer for worker communication
```

Comment:
Both ways are web compatible, it just doesn't work with our current worker implementation. For clarity, I would change this comment to something that better reflects what it does: Calculates and returns a pointer to the lagrange basis